.github/workflows/integration_tests.yml

```diff
@@ -264,9 +245,8 @@ jobs:
                 else
                   echo ":x: Test failed (failed to compile): $filename" >> $GITHUB_STEP_SUMMARY
                   test_err_sum=$((test_err_sum+1))
-                fi ;;
-              *)
```

Comment:
@bettinaheim - this will remove the 🏳️ items (i.e. skipped items) from the test summary. Is that ok with you?

Reply:
True. I don't see a way around it since we'll only run the valid tests.

Reply:
I don't understand why it would be an issue to indicate the skipped tests as such; you will need to make sure that also the tests we skip are included in the general grep pattern, and then match the name of the tests/files that are included.

Reply:
The script changes eliminate the need to hardcode the names of the tests that the nightly testing should run. Instead, there is a directory structure that identifies the total set of nightly tests to be run for each target. This eliminates having to touch the CI every time a nightly test is added and separates out nightly tests from regression tests, instead of conflating the two types of tests.

Reply:
Another option might be a simple change like [this](https://gist.github.com/bmhowe23/ab8af358871099ab78f86e891380ea65). This would:
* Eliminate the duplicate runs in our NVQPP tests
* Requires no changes to the integration_tests.yml file (though we may still want to update it in the future; see below)
* Still has the 🏳️ items in the run summary to indicate that we are skipping specific tests for specific backends.

In the long term (well maybe not tooooo long term as it could probably be done relatively quickly), I'd like to move to a simple text file with a list of tests to run for each backend and have the integration_tests.yml file parse that text file. The text file could also have comments if we disable certain tests for specific reasons. My motivation for a change like this is that I'd rather not have too many symlinks in the repo because they are just a bit onerous grep commands and general maintenance. That is - if people use a simple `grep -R` command, it won't be obvious that the results contain duplicates in them. Also, if we rename any test files in the future, we'd have to remember to update the symlink, too.